regolith/helpers/u_contacthelper.py

```diff
@@ -4,34 +4,33 @@
 import datetime as dt
 from nameparser import HumanName
 import dateutil.parser as date_parser
-import sys
 import uuid
 
 from regolith.helpers.basehelper import DbHelperBase
 from regolith.fsclient import _id_key
-from regolith.tools import all_docs_from_collection
+from regolith.tools import all_docs_from_collection, fragment_retrieval
 
 
 TARGET_COLL = "contacts"
 
 def subparser(subpi):
-    subpi.add_argument("name", help="first name space last name in quotes")
+    subpi.add_argument("name", help="name, name fragment (single argument only) or id "
+                                    "used to find an existing contact. "
+                                    "please, inform full name if this is a new contact")
+    subpi.add_argument("--number", help="number in the numbered list")
```

Comment:
I think this should maybe be called ``--index`` and ``-i`` and help is ``index of the item in the enumerated list chosen to update``

Reply:
done!